spec/features/create_object_h2_spec.rb

```diff
@@ -35,8 +35,14 @@
     # signal that the background processes have finished before trying to
     # deposit an item to a collection.
     reload_page_until_timeout!(text: "Edit #{collection_title}", as_link: true)
+
     # Deposit an item to the collection
-    find(:table, collection_title).sibling('button').click
+    coll_link = find_link(collection_title, match: :first)
```

Comment:
Is it possible to do `find(:table, collection_title).sibling('turbo-frame').child('button').click`?

Reply:
I'll try it, but `.first` hasn't worked in some other contexts.

Reply:
it liked it!   I think John and I got all bolluxed up by turbo-frame itself not being a sibling.  Thanks.